Buddies/Profile/SettingsVC.swift

```diff
@@ -9,12 +9,34 @@
 import UIKit
 import Firebase
 
-class SettingsVC: UIViewController {
+protocol SettingsVCDelegate {
+    func setStarredTopicNotification(to: Bool)
+    func setJoinedActivityNotification(to: Bool)
+}
 
+class SettingsVC: UITableViewController {
+    @IBOutlet weak var joinnedActivityNotificationToggle: UISwitch!
+    @IBOutlet weak var starredTopicNotificationToggle: UISwitch!
+    
+    var delegate: SettingsVCDelegate!
+    var init_TopicsNotifications: Bool!
+    var init_JoinedActivityNotifications: Bool!
```

Comment:
(especially since this is already "joined")

Reply:
I'm sorry, I somehow messed up joined/favorite in my head. I meant that the variable here isn't named "starred"

Reply:
but it would make sense to make this favorite, right?